### PR TITLE
Fixes #15380: Add the User techniques category to the technique packaging

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15380

### CODE REVIEW

File: techniques/ncf_technique/category.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<xml>           
+  <name>User Techniques</name>
```
Comment (amousset) [inline techniques/ncf_technique/category.xml]: This will break if people already have it with a different name IIRC

Comment (peckpeck) [reply techniques/ncf_technique/category.xml]: We do not push categories to existing configuration-repository directory

Comment (amousset) [reply techniques/ncf_technique/category.xml]: won't it be updated by techique-upgrade mechanism?

Comment (peckpeck) [reply techniques/ncf_technique/category.xml]: Then the name will be updated like for other categories.

Comment (amousset) [reply techniques/ncf_technique/category.xml]: the name is not updated when changed in the category file, but a new category is created. This is an old known bug.

Comment (peckpeck) [reply techniques/ncf_technique/category.xml]: %µ$£ù#*

Comment (peckpeck) [reply techniques/ncf_technique/category.xml]: Now it won't anymore

Comment (amousset) [reply techniques/ncf_technique/category.xml]: \o/

**Comment by amousset** [comment]
Needs to be retargeted in 5.1 (and te remove this category creation from the postinstall)

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/rudder-techniques/pull/1503

**Pull request closed without merging**